engine/src/main/java/io/zeebe/engine/processing/bpmn/container/CallActivityProcessor.java

```diff
@@ -165,21 +142,36 @@ public void beforeExecutionPathCompleted(
   @Override
   public void afterExecutionPathCompleted(
       final ExecutableCallActivity element,
-      final BpmnElementContext flowScopeContext,
-      final BpmnElementContext childContext) {}
+      final BpmnElementContext callActivityContext,
+      final BpmnElementContext childContext) {
+    final var completing = stateTransitionBehavior.transitionToCompleting(callActivityContext);
+    onComplete(element, completing);
+  }
```

Comment:
Please consider this change carefully. This implementation means that we directly transition the call activity to completing and completed (while processing the Process:complete records) without a Complete_Element command for the call activity.

Reply:
After discussing, it makes more sense to process complete_element command for call activity. I've changed it.